app/src/main/res/layout/pin_password_activity.xml

```diff
@@ -68,7 +68,8 @@
       android:layout_marginTop="8dp"
       android:cursorVisible="true"
       android:fontFamily="sans-serif"
-      android:inputType="@{viewModel.showPassword ? InputType.TYPE_CLASS_NUMBER : (InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_VARIATION_PASSWORD)}"
+      android:inputType="number"
```

Comment:
update other versions of this file
src/main/res/layout-sw600dp-land/pin_password_activity.xml
src/main/res/layout-sw600dp-port/pin_password_activity.xml
src/main/res/layout-land/pin_password_activity.xml

Reply:
Done